### PR TITLE
Add United Tech domains

### PR DESCRIPTION
**Pull request by Illia-korzh**
### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (`make test`)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the `_psl` TXT record in place in the respective zone(s).

__Submitter affirms the following:__ 
 United Tech is not using this to work around rate-limits of third party services.

 * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

 * [x] This request was _not_ submitted with the objective of working around other third-party limits.

 * [x] The submitter acknowledges that it is their responsibility to maintain the domains within their section. This includes removing names which are no longer used, retaining the _psl DNS entry, and responding to e-mails to the supplied address. Failure to maintain entries may result in removal of individual entries or the entire section.

 * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms to them.
 * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting.

**Abuse Contact:**
info@unitedtech.ai
abuse@unitedtech.ai

* [x] Abuse contact information (email or web form) is available and easily accessible.

  URL where abuse contact or abuse reporting form can be found: 
  By abuse-email message.

---

For PRIVATE section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*. I could break my organization's website cookies and cause other issues, and the rollback timing is acceptable. *Proceed anyways*.
---

## Description of Organization
Our company is engaged in the development of advanced social networks according to the requirements of our clients.

**Organization Website:**
 [https://unitedtech.ai](https://unitedtech.ai) 

## Reason for PSL Inclusion
- Our employees must deploy to Gitlab Pages with a unique subdomain (for our customers and feature-branch environments). 
- We want our tenant customers' apps to be isolated from each other (cookies, etc.)

**Number of users this request is being made to serve:**
3000-5000

## DNS Verification by dig
```
dig +short TXT _psl.pages-unitedtech.link
"https://github.com/publicsuffix/list/pull/2227"
```

```
dig +short TXT _psl.dev-pages-unitedtech.link
"https://github.com/publicsuffix/list/pull/2227"
```

## Results of Syntax Checker (`make test`)
TOTAL: 5
PASS: 5

### CODE REVIEW

File: public_suffix_list.dat [hunk 1]
```diff
@@ -15504,6 +15504,11 @@ unison-services.cloud
 virtual-user.de
 virtualuser.de
 
+// United Tech : https://unitedtech.ai/
+// Submitted by Illia Korzh <illia.k@unitedtech.ai>
+*.pages-unitedtech.link
+*.dev-pages-unitedtech.link
```
Comment (groundcat) [inline public_suffix_list.dat:15510]: Please fix sorting, `dev-pages-unitedtech.link` should be above `pages-unitedtech.link`

File: public_suffix_list.dat [hunk 2]
```diff
@@ -15504,6 +15504,11 @@ unison-services.cloud
 virtual-user.de
 virtualuser.de
 
+// United Tech : https://unitedtech.ai/
+// Submitted by Illia Korzh <illia.k@unitedtech.ai>
```
Comment (groundcat) [inline public_suffix_list.dat:15508]: Please use a non-personal email address.

**Comment by groundcat** [comment]
- [ ]  Expiration (Note: Must STAY >2y at all times)
  - [ ]  pages-unitedtech.link expires 2025-08-29
  - [ ]  dev-pages-unitedtech.link expires 2025-10-18
- [x]  DNS _psl entries (Note: Must STAY in place)
- [ ]  Tests pass
- [ ]  Sorting
- [ ]  Reasoning/Organization description
- [ ]  Non-personal email address
- [ ] Abuse contact

---

- Please correct the sorting by following the guidelines.
- Please extend the registration of your domains to ensure an expiration date more than two years from today. This two-year buffer needs to be maintained continuously in future years to keep the domains in the PSL. Kindly review the guidelines for more information.
- Are you sure you intend to use a wildcard? I assume you don't need the `*.` part for hosting GitLab Pages.
- Please use a non-personal email address so we can contact your organization even if you leave.
- Please review the abuse contact requirements in the PR template and expand on how someone can contact you regarding abuse.
- You mentioned the number of users this request is intended to serve is 250-500—does this refer to the number of GitLab Pages or the number of users who use these pages?
- How many GitLab Pages are currently hosted on the unitedtech domains?
- Is `dev-pages-unitedtech.link` only for the dev environment? If so, it may not meet PSL requirements, and you should only add `pages-unitedtech.link` to the PSL.
- Please note that, according to the [guidelines](https://github.com/publicsuffix/list/wiki/Guidelines#validation-and-non-acceptance-factors):

> Projects that are smaller in scale or are temporary or seasonal in nature will likely be declined. Examples of this might be private-use, sandbox, test, lab, beta, or other exploratory nature changes or requets. It should be expected that despite whatever site or service referred a requestor to seek addition of their domain(s) to the list, projects not serving more then thousands of users are quite likely to be declined.

File: public_suffix_list.dat (hunk 1 above)
Comment (groundcat) [reply public_suffix_list.dat:15510]: The wildcard character matches any valid sequence of characters in a hostname part. I assume you don't need that.

**Comment by Illia-korzh** [comment]
Extended domain registrations:
1 - [https://who.is/whois/dev-pages-unitedtech.link](https://who.is/whois/dev-pages-unitedtech.link)
2 - [https://who.is/whois/pages-unitedtech.link](https://who.is/whois/pages-unitedtech.link)

**Comment by Illia-korzh** [comment]
Change domains from *. to flat.

**Comment by Illia-korzh** [comment]
Submit email fixed

**Comment by Illia-korzh** [comment]
I have clarified the size of the development teams and the potential number of deployments to Pages.
Clarified number: 1700-2500 deployments per month

**Comment by Illia-korzh** [comment]
Nothing is currently hosted. The developers requested this functionality about a month ago.

**Comment by Illia-korzh** [comment]
dev-pages-unitedtech.link is not a dev environment, but rather another tenant.

**Comment by Illia-korzh** [comment]
We have added a secondary abuse email. Determined abuse contacts.

**Comment by wdhdev** [comment]
You note you are attempting to evade Cloudflare and Let's Encrypt limitations, which is not permitted as per our guidelines.

Also, internal-use domains are likely not going to be accepted. Your user count is normally below what we tend to allow as well.

**Comment by Illia-korzh** [comment]
I clarified our business flow with the managers. We are NOT trying to bypass anyone's restrictions, I was wrong in the initial assessment.
The number of users is planned for the first stage. After clarification, the number of users is stated as 3000-5000.

**Comment by wdhdev** [comment]
Your reason for PSL inclusion is very vague however. Why exactly do they need to be listed (e.g. cookie separation)?

**Comment by Illia-korzh** [comment]
Update Reasons

**Comment by groundcat** [comment]
- @Illia-korzh You initially stated that the number of users this request is intended to serve was 250-500, but later updated it to 3000-5000. Could you explain the basis for the 3000-5000 count? Is this the current number of subdomains or registered users? Are these the current users who are using this and may benefit from PSL inclusion, or is this an estimated number of users you expect to have in the future?  
- Using [Certificate Transparency](https://crt.sh/?q=pages-unitedtech.link) for `pages-unitedtech.link`, I did not find any subdomains. I also searched Google and Bing for living examples of subdomains by using `site:pages-unitedtech.link`, but no results came up. The same situation applies to `dev-pages-unitedtech.link`. Do you have a reason why none of the subdomains or any active examples can be discovered?
- Regarding the abuse contact requirement, we need to understand how an internet user can contact you to report abuse. Providing your abuse email address alone to us does not address this question, as the PSL does not handle abuse for you. I checked your organization's website at [https://unitedtech.ai/contacts](https://unitedtech.ai/contacts), but there is only a form to submit job applications. Additionally, the "Contact Us" button in the footer of your website is not functioning. Please walk us through how someone visiting a site like `yourclientname.pages-unitedtech.link`, who encounters abuse, would be able to go from there as a starting point and report such an issue to your organization.

**Comment by simon-friedberger** [comment]
- [x]  Expiration (Note: Must STAY >2y at all times)
  - [x]  pages-unitedtech.link expires 2030-08-29
  - [x]  dev-pages-unitedtech.link expires 2030-10-18
- [x]  DNS _psl entries (Note: Must STAY in place)
- [x]  Tests pass
- [x]  Sorting
- [x]  Reasoning/Organization description
- [x]  Non-personal email address
- [ ] Abuse contact

**Comment by Illia-korzh** [comment]
@groundcat We can create a separate address for processing complaints and add it to the DNS zone information. There are also company contacts on social networks. Which of these will work?

**Comment by simon-friedberger** [comment]
It should be something that people can find on the page when going to dev-pages-unitedtech.link, pages-unitedtech.link and/or unitedtech.ai.

For example, on google.com you click on "Terms" in the footer and search for abuse, for Facebook it's "Imprint/Terms".

RFC 2142 https://www.rfc-editor.org/rfc/rfc2142#section-4 says abuse@ should work.

**Comment by Illia-korzh** [comment]
Sorry for the inconvenience, we have found another way to implement our business objectives. Please close this PR.

**Pull request closed without merging**